matrix-sdk/src/main/java/org/matrix/androidsdk/HomeServerConnectionConfig.java

```diff
@@ -202,6 +232,21 @@ public boolean shouldPin() {
         return mPin;
     }
 
+    public void setAcceptedTlsVersions(List<TlsVersion> tlsVersions) { mTlsVersions = Collections.unmodifiableList(tlsVersions); }
```

Comment:
Please insert new line after `{` (you can format code using Alt+Ctrl+L in Android Studio).

Reply:
Ok